app/models/Quiz.js

```diff
@@ -0,0 +1,34 @@
+const db = require("../config/db");
+
+class Quiz {
+  static save(Question, Answer) {
+    let sql = `INSERT INTO questions (
+                    question,
+                    answer
+                ) VALUES(?, ?)`;
+
+    return db.execute(sql, [Question, Answer]);
+  }
+
+
+  static async findRandomQuestion() {
+    let sql = `SELECT *
+                FROM questions
+                ORDER BY RAND()
+                LIMIT 1`;
+
+    const [question, _] = await db.execute(sql);
+    console.log("Question fetched: ", question);
+    return question[0];
+  }
```

Comment:
Clever way to get a random question, but maybe the model should actually return all questions, and then the controller should select a random one since that is a feature of our internal logic not the db's. We shouldn't leave it to the db model to do it, but what do you think?